architecture/azure/web-queue-worker.md

```diff
@@ -12,18 +12,24 @@ The Azure Architecture Center describes the [web-queue-worker architecture style
 ## Components
 
 * The **web front end** serves client requests, handles authentication and authorization.
-* The front end queues commands for intensive or long-running workloads in the **message queue** for the worker to consume. This allows the web application to remain independent of the worker's workload and to immediately respond to HTTP requests.
-* **The worker** receives work from the message queue. Results can be stored in the data store.
-* Both worker and front end might access the **data store** directly to access business data and process state. An optional **cache** might be used for performance optimizations.
-* A **Content Delivery Network** can help to serve static content even faster and reduce load on the web front end.
+* The front end uses a send-only NServiceBus endpoint to queue commands for intensive or long-running workloads in a **message queue**, such as Azure Service Bus or Azure Storage Queues, for the worker to consume. This allows the web application to remain independent of the worker's workload and to immediately respond to HTTP requests.
+* **The worker**, hosted in services like Azure App Service, Azure Container Apps, Azure Functions, or Virtual Machines, runs a full NServiceBus endpoint that receives work from the message queue. The worker processes messages using business logic and stores results in a data store such as Azure SQL or Cosmos DB.
+* Both worker and front end might access the **data store** directly to access business data and process state. An optional **cache**, such as Azure Cache for Redis, might be used for performance optimizations.
+* A **Content Delivery Network**, like Azure CDN, can help to serve static content even faster and reduce load on the web front end.
```

Comment:
I think the specific technologies aren't mentioned because there is a whole section below that mentions it.